rust/gui-client/src-tauri/src/client/gui.rs

```diff
@@ -3,28 +3,21 @@
 //! Most of this Client is stubbed out with panics on macOS.
 //! The real macOS Client is in `swift/apple`
 
-// TODO: `git grep` for unwraps before 1.0, especially this gui module <https://github.com/firezone/firezone/issues/3521>
-
 use crate::client::{
     self, about, deep_link, logging, network_changes,
     settings::{self, AdvancedSettings},
     Failure,
 };
 use anyhow::{bail, Context, Result};
-use arc_swap::ArcSwap;
-use connlib_client_shared::{file_logger, ResourceDescription, Sockets};
-use connlib_shared::{keypair, messages::ResourceId, LoginUrl, BUNDLE_ID};
+use connlib_client_shared::ResourceDescription;
+use connlib_shared::messages::ResourceId;
```

Comment:
I removed almost all direct references to connlib, but these type definitions are safe to keep even with IPC, so are constants like `BUNDLE_ID`. No point replicating those in `TunnelWrapper` as long as in-process connlib is supported anyway.